src/app-layout/__integ__/app-layout-tools.test.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { BasePageObject } from '@cloudscape-design/browser-test-tools/page-objects';
+import useBrowser from '@cloudscape-design/browser-test-tools/use-browser';
+
+import createWrapper from '../../../lib/components/test-utils/selectors';
+import { viewports } from './constants';
+
+const wrapper = createWrapper().findAppLayout();
+class AppLayoutToolsViewPage extends BasePageObject {
+  async openTools() {
+    await this.click(wrapper.findToolsToggle().toSelector());
+  }
+
+  async closeTools() {
+    await this.click(wrapper.findToolsClose().toSelector());
+  }
+}
+
+describe.each(['classic', 'refresh', 'refresh-toolbar'] as const)('%s', theme => {
+  function setupTest(testFn: (page: AppLayoutToolsViewPage) => Promise<void>, path: string) {
+    return useBrowser(async browser => {
+      const page = new AppLayoutToolsViewPage(browser);
+      const url = `#/light/app-layout/${path}`;
+      await page.setWindowSize(viewports.desktop);
+      const params = new URLSearchParams({
+        visualRefresh: `${theme.startsWith('refresh')}`,
+        appLayoutToolbar: `${theme === 'refresh-toolbar'}`,
+      });
```

Comment:
We have a shared utility for this now: https://github.com/cloudscape-design/components/blob/5b0e892123f96f8004d146d11251dee27894d481/src/app-layout/__integ__/utils.ts#L22-L29